docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,105 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
+
+1. Set up netlify key:
+    * install netlify: `npm install -g netlify-cli`
+    * Navigate to your main directory
+    * Edit your bash profile:
+        * enter `vi ~/.bash_profile` in the command line
+        * Type `o` to enter a new line
+        * Once in insert mode, can copy and paste the following keys, prefixing it with "export":
+            * `export NETLIFY_AUTH_TOKEN= YOURTOKENHERE123`
+            * `export NETLIFY_SITE_ID=cal-itp-data-analyses`
+        * Press `ESC` + `:wq` to stop insert mode and to close the vim
+            * note: `ESC` + `dd` gets rid of unwanted lines
+        * Enter command `source ~/.bash_profile` in the command line.
+        * Enter command: `cat ~/.bash_profile` to see that it's there
+
+2. Create a `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites). Each `.yml` file is a site, so if you have separate research topics, they should each have their own `.yml` file.
+    * This `.yml` file will include the directory to the notebook(s) you want to publish.
+    * Name your `.yml` file
+    * The structure of your `.yml` file depends on the type of your analysis:
+        * If you have one parameterized notebook with **one parameter**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+               - caption: Introduction
+               - chapters:
+                 - params:
+                      district_parameter: 1
+                      district_title: District 1
+            ```
+         * If you have a parameterized notebook with **multiple parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+            - chapters:
+              - caption: County Name
+                params:
+                  paramter1_county_name
+                sections:
+                - city: parameter2_city_name
+                - city: parameter2_city_name
+            ```
+         * If you have an individual notebook with **no parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            parts:
+            - caption: Introduction
+            - chapters:
+              - notebook: ./my-analyses/notebook_1.ipynb
+              - notebook: ./my-analyses/notebook_2.ipynb
+            ```
+## Running the Build Command:
+**Note:** The build command must be run from the root of the repo!
+1. Navigate back to the repo data-analyses and install the portfolio requirements with
+`pip install -r portfolio/requirements.txt`
+2. Then run `python portfolio/portfolio.py build my_report --deploy`
```

Comment:
I would probably recommend running without `--deploy`, and then running with `--no-execute-papermill --deploy`.

Side note, would it be better if build/deploy were entirely separate commands? That might help simplify things.

Reply:
Analysts have been pushing the commits and view the netlify build off of the preview PR URL, since the URL that's holding the content (one akin to https://parallel-corridors--cal-itp-data-analyses.netlify.app) shows that it builds and is deployed, but either is a broken URL or doesn't reflect the changes.

Reply:
Each time I made changes, I just deployed since I was using the [Makefile](https://github.com/cal-itp/data-analyses/blob/main/Makefile). I can change the docs to make the options clearer though!

Reply:
Changed this section a bit- I tried using the `--no-execute-papermill --deploy` on one of my test notebooks and it didn't seem to show any of the changes on the preview url so left that out for now.